.github/workflows/linux.yml

```diff
@@ -132,9 +132,17 @@ jobs:
 
     - name: Measure coverage
       if: ${{ matrix.build-type == 'Coverage' }}
-      timeout-minutes: 5
+      timeout-minutes: 10
+      shell: /bin/bash -xeuo pipefail {0}
```

Comment:
Can revert that now.

Reply:
Oops, right.

Reply:
Ok, done and rebased into a previous related commit.